pkg/subctl/cmd/gather/gather.go

```diff
@@ -185,59 +179,59 @@ func gatherDataByCluster(restConfig restconfig.RestConfig, directory string) {
 			}
 		}
 	}
-	gather.ClusterSummary(info)
+	ClusterSummary(info)
 }
 
-func gatherConnectivity(dataType string, info gather.Info) bool {
+func gatherConnectivity(dataType string, info Info) bool {
 	if info.Submariner == nil {
 		info.Status.QueueWarningMessage("The Submariner connectivity components are not installed")
 		return false
 	}
 
 	switch dataType {
 	case Logs:
-		gather.GatewayPodLogs(info)
-		gather.RouteAgentPodLogs(info)
-		gather.GlobalnetPodLogs(info)
-		gather.NetworkPluginSyncerPodLogs(info)
+		GatewayPodLogs(info)
```

Comment:
Referencing  as `GatewayPodLogs` instead of `gather.GatewayPodLogs` seems awkward. I think we should prepend `gather` to each function name. However I think it's fine to keep this file where it is.

Reply:
Agree on prepending `gather` to function name and also making the functions private.

Just trying to make dir structure clean and consistent by this change.

Reply:
I think it’s interesting to move commands to their own packages: currently the whole `cmd` package is a bit of a mess, and various commands reference each others’ package variables (and sometimes miss important aspects, as happened at one point with contexts etc.). Moving `gather` to its own package is a step in the right direction IMO.